src/Database/Query.php

```diff
@@ -1687,6 +1687,7 @@ public function unionAll($query, $overwrite = false)
      * @param array<int|string, string> $types A map between columns & their datatypes.
      * @return $this
      * @throws \RuntimeException When there are 0 columns.
+     * @deprecated
```

Comment:
If you mark this a deprecated IDEs which show a warning when using `InsertQuery::insert()` too as it inherits this method.